core/src/utilities/helpers/iframe-helpers.js

```diff
@@ -104,3 +104,30 @@ export const getVisibleIframes = () => {
     .call(document.querySelectorAll('iframe'))
     .filter(item => item.style.display !== 'none');
 };
+
+export const getValidIframe = (event, window) => {
+  const iframes = document.querySelectorAll('.iframeContainer iframe');
```

Comment:
Maybe we should not use `querySelectorAll`anymore for iframes, since now we have modal iframes, which are in a different container. Additionally, do we need to consider here also the Luigi Core window in case the message is coming from Luigi Core?
If we should not use that anymore, then we have to remove it also from 'getVisibleIframes` in this file?

Reply:
The way we use it it should rather be isValidMessageSource(e) and should return true if either:
e.source === window
or
there is an iframe in the .iframeContainer where iframe.contentWindow === e.source
or
we have modalIframe and modalIframe.contentWindow === e.source.